src/code42cli/cmds/securitydata.py

```diff
@@ -470,74 +457,49 @@ def show(state, search_id):
     echo(pformat(response["searches"]))
 
 
-@security_data.command(cls=SendToCommand)
-@file_event_options
-@search_options
-@click.option(
-    "--or-query",
-    is_flag=True,
-    cls=searchopt.AdvancedQueryAndSavedSearchIncompatible,
-    help="Combine query filter options with 'OR' logic instead of the default 'AND'.",
-)
-@sdk_options()
-@server_options
-@click.option(
-    "--include-all",
-    default=False,
-    is_flag=True,
-    help="Display simple properties of the primary level of the nested response.",
-)
-@send_to_format_options
-def send_to(
-    state, begin, end, advanced_query, use_checkpoint, saved_search, or_query, **kwargs,
-):
-    """Send events to the given server address.
+def _get_file_event_cursor_store(profile_name):
+    return FileEventCursorStore(profile_name)
 
-    HOSTNAME format: address:port where port is optional and defaults to 514.
-    """
-    cursor = _get_cursor(state, use_checkpoint)
 
-    if use_checkpoint:
-        checkpoint_name = use_checkpoint
-        # if checkpoint name exists, checkpoint should be that eventId,
-        # otherwise it should set the initial value to ""
-        checkpoint = cursor.get(checkpoint_name) or ""
-
-        # older app versions stored checkpoint as float timestamp.
-        # we handle those here until the next run containing events will store checkpoint as the last eventId
-        try:
+def _construct_query(state, begin, end, saved_search, advanced_query, or_query):
+
+    if advanced_query:
+        state.search_filters = advanced_query
+    if saved_search:
```

Comment:
I don't think this really matters (I wrote it actually I think) - but since exactly one of these should be true at a time, for consistency this could probably be `elif` or all three cases could be their own `if`? 

I think `advanced-search`, `saved-search` and `begin/end` are all exclusive from each other

Reply:
yeah, this is just showing up in the diff cause I moved things around a bit. Although an `elif` makes it a bit more explicit that these states shouldn't ever overlap, I'll update it.